var/spack/repos/builtin/packages/gpgme/package.py

```diff
@@ -13,8 +14,16 @@ class Gpgme(AutotoolsPackage):
     homepage = "https://www.gnupg.org/software/gpgme/index.html"
     url      = "https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.12.0.tar.bz2"
 
+    executables = ['^gpgme-config$']
+
     version('1.12.0', sha256='b4dc951c3743a60e2e120a77892e9e864fb936b2e58e7c77e8581f4d050e8cd8')
 
     depends_on('gnupg', type='build')
     depends_on('libgpg-error', type='build')
     depends_on('libassuan', type='build')
+
+    @classmethod
+    def determine_version(cls, exe):
+        output = Executable(exe)('--version', output=str, error=str)
+        match = re.search(r'(\S+)', output)
+        return match.group(1) if match else None
```

Comment:
```suggestion
        return Executable(exe)('--version', output=str, error=str).rstrip()
```
This is how I do it in the GDAL package.

Reply:
Thanks, that is much easier.